### PR TITLE
add support for older supermicro redfish model motherboards

### PR DESCRIPTION
**Pull request by nsatsia**
Add a translation of Redfish API supported by older Supermicro models, I believe the Supermicro redfish v1 API.

### CODE REVIEW

File: supermicro_scripts/Containerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+FROM registry.fedoraproject.org/fedora:35
+MAINTAINER Mario Vazquez <mavazque@redhat.com>
+
+RUN echo -ne "[racadm]\nname=Racadm\nbaseurl=http://linux.dell.com/repo/hardware/dsu/os_dependent/RHEL8_64\nenabled=1\ngpgcheck=0" > /etc/yum.repos.d/racadm.repo
```
Comment (mvazquezc) [inline supermicro_scripts/Containerfile:4]: This shouldn't be required for SuperMicro.

File: supermicro_scripts/Containerfile [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+RUN set -x && \
+    dnf -y update && \
+    dnf install -y python3 python3-flask python3-requests python3-pyOpenSSL openssl-devel srvadmin-idracadm7 && \
```
Comment (mvazquezc) [inline supermicro_scripts/Containerfile:8]: srvadmin-idracadm7 not needed here.

**Review by mvazquezc — COMMENTED** [review]
Just a couple of comments.

File: supermicro_scripts/Containerfile (hunk 1 above)
Comment (nsatsia) [reply supermicro_scripts/Containerfile:4]: Sorry, ignore the Containerfile I just copied the whole directory dell_scripts.

The motherboard I've been testing with in my lab is a workstation one, model X11scz-f

**Comment by mvazquezc** [comment]
lgtm, maybe squash the commits. I don't have rights to merge it, so we need to wait for @dtantsur

**Pull request merged**